website/docs/cloud-docs/api-docs/workspaces.mdx

```diff
@@ -2789,13 +2789,14 @@ Status code `204`.
 
 The GET endpoints above can optionally return related resources, if requested with [the `include` query parameter](/terraform/cloud-docs/api-docs#inclusion-of-related-resources). The following resource types are available:
 
-* `organization` - The full organization record.
-* `current_configuration_version` - The last configuration this workspace received, excluding plan-only configurations. Terraform uses this configuration for new runs, unless you provide a different one.
 * `current_configuration_version.ingress_attributes` - The commit information for the current configuration version.
-* `current_run` - Additional information about the current run.
-* `current_run.plan` - The plan used in the current run.
-* `current_run.configuration_version` - The configuration used in the current run.
+* `current_configuration_version` - The last configuration this workspace received, excluding plan-only configurations. Terraform uses this configuration for new runs, unless you provide a different one.
 * `current_run.configuration_version.ingress_attributes` - The commit information used in the current run.
+* `current_run.configuration_version` - The configuration used in the current run.
+* `current_run.plan` - The plan used in the current run.
+* `current_run` - Additional information about the current run.
```

Comment:
(Relates to my below comment)
It feels like `current_run` should also be above all of the other attributes so folks have context on what `current_run` is.

Reply:
Will rollup this comment to the one above